packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -303,19 +342,22 @@ class _CpuFlameChart extends CoreElement {
 
   CoreElement stackFrameDetails;
 
-  CoreElement error;
-
   CpuProfileData cpuProfileData;
 
   TimelineEvent event;
 
+  /// Stores the latest timeline snapshot.
+  ///
+  /// This will be null if a cpu profile was not loaded from snapshot.
+  TimelineSnapshot snapshot;
```

Comment:
instead of storying the snapshot here, we should use this as a chance to refactor this code to have more of a data model and ui separation so that display from a snapshot invloves using a data model from a different source and minimal changes to code that is just about ui display.